src/promptflow/promptflow/_sdk/data/executable/main.py.jinja2

```diff
@@ -159,8 +159,8 @@ def start():
             else:
                 os.chdir(flow.parent)
             invoker = FlowInvoker(flow, connection_provider="local", dump_to=dump_path)
-        result = invoker.invoke(data)
-        print_yellow_warning(f"Result: {result}")
+        result, result_output = invoker.invoke(data)
+        print_yellow_warning(f"Result: {result_output}")
```

Comment:
move the print into flow_invoker is ok, then we could make no change to function return values.

Reply:
updated